### PR TITLE
Projects Redesign

### LINKED ISSUE #34: Redesign project page
Todo:
- [x] Submit design team ticket
- [ ] Do things
- [x] Ask teams for project summaries + media

### LINKED ISSUE #46: Feature interesting presentations, demos, and Medium posts
Jordan's [blockchain presentation](https://docs.google.com/presentation/d/1m_QU1NWzE0yPk7wVUEqBBn_FCTvB4kt52XUupjMihEw/edit#slide=id.g358057f5c5_0_157), for example, and some of the demos too I think!

Also a section featuring the [Medium publication](https://medium.com/ubc-launch-pad-software-engineering-blog) would be cool too - this might all help give the impression that LP projects are typically more meaningful than the average personal project

### PR DESCRIPTION
**Pull request by bobheadxi**
Closes #34 
Closes #46 

image

Preview: https://deploy-preview-50--ubclaunchpad.netlify.com

*[Update]* See: https://github.com/ubclaunchpad/ubclaunchpad.github.io/pull/50#issuecomment-388657905

---

Redesigned project section on homepage
Moved "About" section to the top
Lots of CSS adjustments, etc - fixed the column spacing in the "About" section
Updated links - new links for various projects (slides/medium posts), switched medium link in footer to publication link
Changed some media - notably Cumulus and Eat (eat's iPhone X screenshot was too tall)

To Do:
- [x] Make projects section scrollable - WIP: ~leftmost elements can't be reached for some reason rip~ couldn't get pure CSS working, see other comment
- [x] Images... don't seem vertically aligned?
- [x] Migrate all projects to homepage
       - didn't add Sift because there's no media, and the web app isn't very screenshotable - if anyone has any images that would help 💭 
- [x] Possibly remove the separate Project page altogether - almost, "See All Projects" now points to GitHub instead
- [x] ~Make all project cards the same height~ meh

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -59,86 +59,343 @@ <h1>
...
         </div>
-        <h2>PLATFORM TEAMS</h2>
         <p class="small">
-          Our platform teams dive deep into their platform of choice and develop apps from conception to release. We have teams for
-          iOS, Android, Web, and Machine Learning.
+          Sleuth is a search engine designed to automatically collect and provide tailored UBC-related content that are relevant to UBC students,
+          such as course sites, news articles, relevant social media content, and more. The website displays the results in a graph view that visualizes
+          the relations between results. Sleuth was built with React, Django, and Apache Solr, and uses Scrapy for content curation.
         </p>
+        <div class="project-links">
```
Comment (jordanschalm) [inline index.html]: It looks a bit weird IMO to have two GH icons and links. I would have only one, pointing to the "main" repo, and make sure that repo references any other repos in a project.

**Review by jordanschalm — APPROVED** [review]
Thanks for doing this @bobheadxi, looks great! 😀 

Looks ready to go besides the bug with the left-most project card.

File: index.html (hunk 1 above)
Comment (bobheadxi) [reply index.html]: that's a good point

**Comment by bobheadxi** [comment]
Couldn't get pure CSS working 😢 Used this library in the end: https://github.com/kenwheeler/slick

It does require jquery 😓  but it's pretty nice and not slow, and has a lot of options and whatnot. Autoplays, pauses on hover, swipe-able, has forward/back buttons.

Low quality gif:

project-demo-full

everything works on mobile too (set to autoscroll faster on mobile):

projects-demo

**Review by bfbachmann — COMMENTED** [review]
This is excellent!! Thanks a ton.

My only comments are:
1. Can you slow down that auto-scroll on the projects section to about once ever 3 seconds? I think it's a bit too fast at the moment.
2. I guess it's okay to put the minified Slick content in the repo but it might be cleaner to just use a CDN if we can, like we do the our other dependencies.

**Review by bfbachmann — APPROVED** [review]
👍

**Comment by bobheadxi** [comment]
@bfbachmann Good point - I've removed the minified Slick library.

I've also decreased the speed for the autoplay on desktop + tablet-sized screens. Desktop is now at 3s, but I kept the tablet-size and mobile-size autoplay faster (2s and 1s respectively) because at those sizes it's not as obvious that it's a carousel, and it's super easy to stop the autoplay when you touch it as you scroll on a touch screen.

**Comment by bfbachmann** [comment]
@bobheadxi You're just too clean!

**Pull request merged**